src/metabase/pulse/render.clj

```diff
@@ -398,8 +398,10 @@
 
 (defn render-pulse-card
   "Render a single CARD for a `Pulse`. DATA is the `:data` from QP results (I think)."
-  [card data]
+  [card result]
   (try
+    (if (:error result) (throw (Exception. "Card has errors")))
```

Comment:
In Lisp languages it's idiomatic to only use `if` if there's an else clause.

```clojure
;; good
(if condition
  (do-something)
  (do-something-else))

;; bad
(if condition
  (do-something))
```

Use `when` if there's no else clause.

```clojure
(when some-condition
  (do-something))
```

Also the `do-something` part should be on a new line.